tests/filecheck/cmath.xdsl

```diff
@@ -0,0 +1,36 @@
+// RUN: xdsl-opt %s | xdsl-opt | filecheck %s
```

Comment:
So this file is the cmath dialect, but written as a program with the IRDL dialect.
Could you remove this file for a subsequent PR? And instead have a file that test CMath.
So for instance something like https://github.com/opencompl/dyn-dialect/blob/main/test/Dyn/test-cmath.mlir, but using an xDSL syntax